packages/salesforcedx-utils-vscode/src/cli/deployResultParser.ts

```diff
@@ -6,65 +6,71 @@
  */
 import { EOL } from 'os';
 
-export interface DeployError {
+export const CONFLICT_ERROR_NAME = 'sourceConflictDetected';
+
+export interface DeployResult {
```

Comment:
A conflict has the same fields as a successful deploy. I just decided to make everything a "DeployResult"